pkg/util/ec2/ec2_tags.go

```diff
@@ -151,6 +169,8 @@ func fetchTagsFromCache(ctx context.Context) ([]string, error) {
 		return nil, fmt.Errorf("unable to get tags from aws and cache is empty: %s", err)
 	}
 
+	tags = filterExcludedTags(tags)
```

Comment:
Missing `filterExcludedTags` function?

Reply:
ohh you're right ! It was on the version prior to the force push iirc
I didn't pay attention mb 🤦‍♂️

Reply:
missed this in the merge, sorry